packages/netlify-cms-locales/src/en/index.js

```diff
@@ -157,11 +157,15 @@ const en = {
       },
       image: {
         choose: 'Choose an image',
+        chooseUrl: 'Insert from URL',
+        promptUrl: 'Please enter URL for your image.',
```

Comment:
Can we make this consistent with `linkPrompt`, e.g. `Enter the URL of the image`?

Reply:
Sure! Do you want to change the property name to be consistent with `linkPrompt` too?